encoding.go

```diff
@@ -64,11 +64,17 @@ func ParseNode(serialized []byte, depth, width int) (VerkleNode, error) {
 		if err != nil {
 			return nil, err
 		}
-		value, _, err := rlp.SplitString(rest)
-		if err != nil {
+		var values [][]byte
+		if err := rlp.DecodeBytes(rest, &values); err != nil {
```

Comment:
Hm, I'm guessing more of this `ParseNode` fn can be re-written to use `DecodeBytes` in future